packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -467,3 +467,38 @@ export class Store {
     return store.state[facetType];
   }
 }
+
+export function AriaLiveRegion(regionName, elem, assertive = false) {
+
+  function dispatchMessage(message) {
+    // Creates the event to dispatch an event to an aria-live region
+    const registerRegionEvent = new CustomEvent('arialivemessage', {
```

Comment:
I think this event should be named `ariaLiveMessageEvent` instead.
```suggestion
    const ariaLiveMessageEvent = new CustomEvent('arialivemessage', {
```

Reply:
The conventions wants event names to be all lowercase. The reason seems to be because attribute names are case insensitive but even names are case sensitive so it can create some conflicts.

But I could use kebab-case if you don't like the name?

Reply:
I misread your message, yes that was a copy-paste error :)